### PR TITLE
[master][KOGITO-1770] Renaming leftovers on README

### PR DESCRIPTION
**Pull request by ricardozanini**
See:
https://issues.redhat.com/browse/KOGITO-1770

Leftovers after merges and rebases

### CODE REVIEW

**Review by cristianonicolai — APPROVED** [review]

**Review by jstastny-cz — CHANGES_REQUESTED** [review]
Hey, just checked <name> tags in each module by running:
```
git grep -i "<name>Kogito example.*kogito.*</name>" -- **/pom.xml
```
And those still look like not touched during the changes:
```
process-business-rules-quarkus/pom.xml:  <name>Kogito Examples :: Kogito Business Rules Quarkus</name>
process-business-rules-springboot/pom.xml:  <name>Kogito Examples :: Kogito Business Rules Spring Boot</name>
process-scripts-springboot/pom.xml:  <name>Kogito Examples :: Kogito Scripts With Spring Boot</name>
process-service-calls-springboot/pom.xml:  <name>Kogito Examples :: Kogito Service Calls with Spring Boot</name>
process-springboot-example/pom.xml:  <name>Kogito Example :: Kogito and SpringBoot</name>
```

**Review by evacchi — APPROVED** [review]

**Review by radtriste — APPROVED** [review]

**Comment by ricardozanini** [comment]
> Hey, just checked tags in each module by running:
> 
> ```
> git grep -i "<name>Kogito example.*kogito.*</name>" -- **/pom.xml
> ```
> 
> And those still look like not touched during the changes:
> 
> ```
> process-business-rules-quarkus/pom.xml:  <name>Kogito Examples :: Kogito Business Rules Quarkus</name>
> process-business-rules-springboot/pom.xml:  <name>Kogito Examples :: Kogito Business Rules Spring Boot</name>
> process-scripts-springboot/pom.xml:  <name>Kogito Examples :: Kogito Scripts With Spring Boot</name>
> process-service-calls-springboot/pom.xml:  <name>Kogito Examples :: Kogito Service Calls with Spring Boot</name>
> process-springboot-example/pom.xml:  <name>Kogito Example :: Kogito and SpringBoot</name>
> ```

Cool thanks!

**Review by MarianMacik — COMMENTED** [review]
I found one more thing which is already fixed on 0.9.x.
https://github.com/kiegroup/kogito-examples/blob/132ec897644e7fdd49a00859de6019681b3b99c2/process-springboot-example/pom.xml#L12

**Comment by ricardozanini** [comment]
Leftovers fixed. @jstastny-cz @MarianMacik could you take a final look please?

**Comment by spolti** [comment]
Is this PR ready to merge?

**Comment by ricardozanini** [comment]
@spolti tomorrow @jstastny-cz and @MarianMacik will run their final check.

**Review by jstastny-cz — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -18,82 +18,82 @@ Everyone is encouraged to contribute to these examples by
...
 
-## Kogito with user tasks with security on REST api
+## Process with user tasks with security on REST api
 
 shows user task interactions with four eye principle applied with security restrictions on REST api.
 
-* [on Quarkus](kogito-usertasks-with-security-quarkus)
-* [on SpringBoot](kogito-usertasks-with-security-springboot)
+* [on Quarkus](process-usertasks-with-security-quarkus)
+* [on SpringBoot](process-usertasks-with-security-springboot)
```
Comment (jstastny-cz) [inline README.md]: SpringBoot here too

File: process-springboot-example/pom.xml [hunk 2]
```diff
@@ -9,7 +9,7 @@
   <groupId>org.kie.kogito.examples</groupId>
   <artifactId>process-springboot-example</artifactId>
 
-  <name>Kogito Example :: Kogito and SpringBoot</name>
+  <name>Kogito Example :: Process and SpringBoot</name>
```
Comment (jstastny-cz) [inline process-springboot-example/pom.xml]: SpringBoot here still

**Review by jstastny-cz — APPROVED** [review]

**Pull request merged**